tests/unit/blocks/block-registration.test.js

```diff
@@ -0,0 +1,247 @@
+/**
+ * @jest-environment jsdom
+ */
+import 'jest-expect-message'
+import { getAttributes } from '../../../scripts/editor/register-blocks';
+import wrapperManifest from '../../../blocks/init/src/Blocks/wrapper/manifest.json';
+import globalManifest from '../../../blocks/init/src/Blocks/manifest.json';
+import { blockAttributes } from '../../data/block-attributes';
+import { expect } from '@jest/globals';
+import { getAllComponentManifests, getAllBlockManifests, getComponentDependencies, recursiveBuildProps, getBlockDependencies, getComponentManifest, getBlockManifest, getMockComponentManifest, getMockBlockManifest } from '../../helpers/blocks';
+import { propsOutput } from '../../data/props-output';
+
+/**
+ * Returns the name of the component from builtProps.
+ *
+ * @param {object} builtProps Built props for a component
+ *
+ * @returns {string}
+ */
+const debugName = (builtProps) => {
+	return `"${builtProps.realName}" (newName: "${builtProps.newName || builtProps.realName}") component`;
+}
+
+/**
+ * Asserts that built props match the expected output.
+ *
+ * @param {object} expectedProps propsOutput object for specific component
+ * @param {object} builtProps Props built for this object.
+ */
+const recursiveAssertProps = (expectedProps, builtProps, isBlock = false) => {
+
+	// Some sanity check
```

Comment:
Intended comment or? Dot at the end.

```suggestion
	// Some sanity check
```